source/includes/administration/_usage.md

```diff
@@ -0,0 +1,69 @@
+## Usage summary
+
+<!-------------------- LIST USAGE SUMMARY -------------------->
+
+### List usage summary
+
+`GET /usage_summary/organizations/:id`
+
+```shell
+# Retrieve usage summary in JSON
+curl "https://cloudmc_endpoint/v1/usage_summary/organizations/03bc22bd-adc4-46b8-988d-afddc24c0cb5?start_date=2017-05-01&end_date=2017-05-15&format=json" \
+   -H "MC-Api-Key: your_api_key"
+
+# Response body example
+```
+```json
+{
+  "data": [{
+    "organizationId": "52fd201e-aa82-4a27-86b3-ea9650a7fb1e",
+    "serviceConnectionId": "beeba736-0451-49b0-8020-8b93ed5abb35",
+    "serviceConnectionPricingId": "e37cc44a-47b6-4a26-81f5-1dbf85433e36",
```

Comment:
I mean, why are we returning it? Can end users query service connection pricings?